<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addMember</class>
 <widget class="QDialog" name="addMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(223, 233, 239);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="btnExitAddMember">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>310</y>
      <width>73</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="memberName_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMemberName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNO_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLabel" name="Addresstxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonYES_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailtxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="phoneNumber_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Phone reminders:</string>
    </property>
   </widget>
   <widget class="QLabel" name="phone_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mobile Phone:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveMember">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>73</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMemberAddress">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMemberPhone">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMemberEmail">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LibraryTitle_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>209</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(40, 83, 107, 1);
font: 75 9pt &quot;Yu Gothic UI&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>ADD NEW MEMBER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
